### PR TITLE
 docs:  update docs to reflect the changes in RxJS 6

### PR DESCRIPTION
**Pull request by siddhant1**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ X] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Rxjs `of` and `from` are documented according to previous version of rxjs

Issue Number: N/A

## What is the new behavior?
The docs are updated to reflect changes in rxjs6

## Does this PR introduce a breaking change?
```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by siddhant1** [comment]
I have signed it

**Comment by siddhant1** [comment]
ref #26182

File: aio/content/guide/observables.md [hunk 1]
```diff
@@ -36,10 +37,10 @@ An `Observable` instance begins publishing values only when someone subscribes t
 
 <div class="alert is-helpful">
 
-   In order to show how subscribing works, we need to create a new observable. There is a  constructor that you use to create new instances, but for illustration, we can use some static methods on the `Observable` class that create simple observables of frequently used types:
+In order to show how subscribing works, we need to create a new observable. There is a constructor that you use to create new instances, but for illustration, we can use some static methods from the `rxjs` library that create simple observables of frequently used types:
```
Comment (trotyl) [inline aio/content/guide/observables.md]: Also `static methods` doesn't fit any more, they're just plain functions now.

Comment (siddhant1) [reply aio/content/guide/observables.md]: I have to just remove the static keyword? Any other changes that I have to make?

**Comment by sarunint** [comment]
The commit message should be `docs: update docs to reflect the changes in RxJS 6`.

Hint: The spaces are important, and you should use present verb.

**Comment by siddhant1** [comment]
Thanks for picking that out !

File: aio/content/guide/observables.md (hunk 1 above)
Comment (trotyl) [reply aio/content/guide/observables.md]: It's no longer `method`, just `function`.

**Comment by sarunint** [comment]
If possible, please squash all commits together and rewrite the commit message.

**Review by jenniferfell — APPROVED** [review]

**Comment by jenniferfell** [comment]
@jasonaden @benlesh This lgtm from an editorial POV. Needs your review/approval from the technical side. Is this what we want to say now? Thanks!

File: aio/content/guide/observables.md [hunk 2]
```diff
@@ -36,10 +37,10 @@ An `Observable` instance begins publishing values only when someone subscribes t
 
 <div class="alert is-helpful">
 
-   In order to show how subscribing works, we need to create a new observable. There is a  constructor that you use to create new instances, but for illustration, we can use some static methods on the `Observable` class that create simple observables of frequently used types:
+In order to show how subscribing works, we need to create a new observable. There is a constructor that you use to create new instances, but for illustration, we can use some methods from the `rxjs` library that create simple observables of frequently used types:
```
Comment (brandonroberts) [inline aio/content/guide/observables.md]: `rxjs` -> `RxJS`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/guide/observables.md (hunk 2 above)
Comment (jenniferfell) [reply aio/content/guide/observables.md]: I checked in the current RxJS docs, and it looks like we use this convention:
- In flowing text, "RxJS does things" without code font
- In code examples or other places where we explicitly name constructs, "rxjs" lowercase

https://angular.io/guide/rx-library

To me, this feels like the second case, where we explicitly name the library and use the lowercase. 

Question: Does the library self-identify as lowercase or mixed case? If code references should be mixed case, then we should open an issue to fix other docs, too. :-)

**Review by brandonroberts — APPROVED** [review]

File: aio/content/guide/observables.md (hunk 2 above)
Comment (brandonroberts) [reply aio/content/guide/observables.md]: The library identifies as mixed case. Its used as lowercase when used in code reference for imports. I think it should be used here as mixed case, but we can omit the code font.

**Comment by brandonroberts** [comment]
@siddhant1 one more thing. Will you remove the backticks from RxJS and squash everything down to one commit?

**Comment by siddhant1** [comment]
Sure

**Comment by brandonroberts** [comment]
@jenniferfell the failure was a flaky test with Travis and unrelated to these changes. I restarted the test and it passed.

**Pull request closed without merging**